### PR TITLE
add structureharvester

### PR DESCRIPTION
**Pull request by lecorguille**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/structureharvester/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/*.py $PREFIX/bin && chmod +x $PREFIX/bin/structureHarvester.py
```
Comment (johanneskoester) [inline recipes/structureharvester/build.sh]: Does this mean that other (non-executable) python files will pollute $PREFIX/bin? If so, please put them into $PREFIX/lib/structureharvester instead, and have a bash wrapper or symlink that points to the main python script.

File: recipes/structureharvester/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+{% set version = "0.6.94" %}
+{% set commit = "e15952601b946267ce566df6ff0dd56df5c8b8f6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.zip
+  md5: 41ce1a230ba9a4ee9b7dca2dd63cc700
```
Comment (johanneskoester) [inline recipes/structureharvester/meta.yaml]: Please use sha256.

File: recipes/structureharvester/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% set name = "structureHarvester" %}
+{% set version = "0.6.94" %}
+{% set commit = "e15952601b946267ce566df6ff0dd56df5c8b8f6" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (johanneskoester) [inline recipes/structureharvester/meta.yaml]: Please put the name directly in here. Our goal is to only use jinja for stuff that occurs more than once and changes upon new releases.

Comment (lecorguille) [reply recipes/structureharvester/meta.yaml]: Done

File: recipes/structureharvester/meta.yaml (hunk 2 above)
Comment (lecorguille) [reply recipes/structureharvester/meta.yaml]: Done

File: recipes/structureharvester/build.sh (hunk 1 above)
Comment (lecorguille) [reply recipes/structureharvester/build.sh]: There is only on other file.py which indeed isn't executable.
But sorry, I'm not sure to understand this alternative:
`symlink that points to the main python script.`
Do you have an example in mind?

**Pull request merged**